### DISCUSSION TITLE
many broken links

### DISCUSSION THREAD

**Discussion by Rustimation** [philippoo66/optolink-splitter#39] [General]
Hi Phil,
gefühlt ist jedes zweite Link in deiner Github Page kaputt. Liegt wohl daran, dass du das Wiki schön neu strukturiert hast. Klar, kann man damit umgehen, schöner weäre es natürlich, wenn alle Links funktionieren würden.
Viele Grüße
Chris

**Comment by philippoo66** [comment]
Hai Chris!

ich hab im Readme _einen_ gefunden (command syntax) und gefixt.

bitte hilf mir - wo sind noch welche?

**Reply by Rustimation** [reply]
habe per Mail geantwortet.

**Comment by Rustimation** [comment]
Okay, hier als Diskussionsbeitrag: Mails an Phil scheinen in der Bit Bucket zu landen... ;-)
Readme Seite: [Kapitel command syntax MQTT, TCP/IP requests](https://github.com/philippoo66/optolink-splitter?tab=readme-ov-file#command-syntax-mqtt-tcpip-requests) --> Link bei **Details see here**

im[ Command Syntax Wiki ](https://github.com/philippoo66/optolink-splitter/wiki/010-Command-Syntax) führen praktisch alle Links hinter "see here" in Leere

**Reply by philippoo66** [reply]
moin Chris!

ich hoffe ich hab sie jetzt alle... Im Wiki kannst du die Links übrigens auch selbst reparieren, das ist offen.

Welche eMail Adresse hast du benutzt? ich meine öffentlich wäre nur die open3e at web. bei mir ist tatsächlich nix angekommen...

**Reply by Rustimation** [reply]
Hallo Phil,
mein Fehler. Ich hatte im Github Benachrichtigungsmail einfach auf "Reply" geclickt und dabei übersehen, dass notifications@github.com dir wohl nicht zugeordnet wird. Sollte ich im Wiki noch was finden, bereinige ich das dann.
VG
Chris
